### DISCUSSION TITLE
Clean Snapshot via API?

### DISCUSSION THREAD

**Discussion by leadZERO** [blakeblackshear/frigate#7928] [Ask A Question] [answered]
Is it possible to get the clean snapshot via the API?

R

**Comment by NickM-27** [comment, marked as answer]
You can get it via the file path `http://frigate_ip:5000/clips/<event_id>_clean.png` but there is currently no api like the snapshot one

**Reply by leadZERO** [reply]
Just to update this for those that come after...

URL is: 'http://{frigate_server}:{frigate_port}/clips/{match["camera"]}-{match["id"]}-clean.png'
